torchvision/datasets/_optical_flow.py

```diff
@@ -180,7 +180,7 @@ class KittiFlow(FlowDataset):
 
     _has_builtin_flow_mask = True
 
-    def __init__(self, root, split="train", transforms=None):
+    def __init__(self, root, split="train", transforms=None) -> None:
```

Comment:
Same flows for all comments

```suggestion
    def __init__(self, root: str, split: str = "train", transforms: Optional[Callable] = None) -> None:
```